src/controllers/OrganisationsController.js

```diff
@@ -140,7 +156,15 @@ const organisationsController = {
 
       const issues = await performanceDbApi.getLpaDatasetIssues(lpa, datasetId)
 
-      const taskList = performanceDbApi.getTaskList(issues)
+      const taskList = issues.map((issue) => {
+        return {
+          title: {
+            text: performanceDbApi.getTaskMessage(issue.issue_type, issue.num_issues)
+          },
+          href: `/organisations/${lpa}/${datasetId}/${issue.issue_type}`,
+          status: getStatusTag(issue.status)
+        }
+      })
```

Comment:
moved this from performanceDbApi to here as the performance db api wont be responsible for this and also it is only used in this method